docs/getting_started.md

```diff
@@ -237,9 +237,8 @@ option = st.sidebar.selectbox(
 'You selected:', option
 ```
 
-Most of the elements you can put into the main part of your app with an
-`st.something()` call can also be put into a sidebar with
-`st.sidebar.something()`. The only exceptions right now are `st.write` (you
+Most of the elements you can put into your app can also be put into a sidebar with
+`st.sidebar.[element_name]()`. The only exceptions right now are `st.write` (you
```

Comment:
I wonder if the square brackets won't be confusing to some people...

Reply:
In code, I've seen the following conventions as well:

* `st.sidebar.<replace_with_element>()`
* `st.sidebar.{{element_name}}()`

This might read cleaner if we left it as is, then immediately provided an example: 

> Most of the elements you can put into your app can also be added to the sidebar with `st.sidebar.<element_name>()`. For example, `st.element.checkbox()`.

Reply:
I don't particularly like the curly brackets -- they look a little too much like params.

Reply:
How about saying something like "where `[element_name]` is a name like `line_chart`"?

Or maybe give examples:

> Most of the elements you can put into your app can also be put into a sidebar with
`st.sidebar.[element_name]()`. For example, `st.sidebar.markdown`, `st.sidebar.slider`, `st.siderbar.line_chart`.